packages/aft/pubspec.yaml

```diff
@@ -11,24 +11,30 @@ dependencies:
   async: ^2.8.0
   aws_common:
     path: ../aws_common
+  built_collection: ^5.0.0
+  built_value: ">=8.4.0 <8.5.0"
   checked_yaml: ^2.0.0
   cli_util: ^0.3.5
   collection: ^1.16.0
   flutter_tools:
     git:
       url: https://github.com/flutter/flutter.git
-      ref: f186e1bd3ed0c8471ad9b52929f70292621ab796
+      ref: 117a83a4a7178c328d296748bd93ff388724cb67
```

Comment:
I had to change this to `7a743c8816fd8ff7df99858c545c1dbe396d1103` in my env to run due to error, dk if anyone else has this issue.

Reply:
What was the error? I've updated it but I haven't noticed any issues locally or in CI.

Reply:
I don't have the error off hand and it was a few weeks ago but it was a version incompatibility thing and the ref I mentioned updated the version to be the one that matches other requirement. https://github.com/flutter/flutter/commit/7a743c8816fd8ff7df99858c545c1dbe396d1103#diff-a3d4dfa7e9543c501bc1aee57d5502b657d6f5469c09fb42b8c9d516ec53df66, maybe frontend_server_client.